Sources/ForageSDK/Foundation/Telemetry/ForageLogger.swift

```diff
@@ -116,6 +119,8 @@ internal class DatadogLogger : ForageLogger {
             in: datadogInstance
         )
         
+        self.logger?.addAttribute(forKey: "trace_id", value: self.config?.traceId)
```

Comment:
we can instead pass `traceId` via the `ForageLoggerContext` object, which will set it when we call addContext [here](https://github.com/teamforage/forage-ios-sdk/pull/95/files#diff-e079539e1d6fdb51dad7a4f72feeb7c0b00815118a211e5a5480af80314a9732R124)

```swift
// in ForageSDK.swift

// in ForageSDK.swift
   let logger = DatadogLogger(
            ForageLoggerConfig(
                environment: self.environment,
                prefix: "",
                context: ForageLogContext(
                    traceId: generateTraceId()
                )
            )
        )
```

So we can safely 
- remove the new line you've added on 122.
- remove `traceid` as a param in the ctor of `ForageLogger
- remove `traceId` as a member of ForageLogger